__tests__/supertest.test.js

```diff
@@ -91,3 +91,44 @@ describe("GET /api/reviews", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  it("should return an array of comment objects for the given review_id with the following properties: comment_id, votes, created_at, author, body, review_id", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then((res) => {
+        const comments = res.body.comments;
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            review_id: expect.any(Number),
+          });
+        });
+        expect(comments).toBeSorted("created_at", {
+          descending: true,
+        });
+        expect(comments.length).toBe(3);
+      });
+  });
+  it("comments don't exist for given review id, responds with a 404", () => {
```

Comment:
Technically this situation isn't an error if we think about it, the review_id is valid and it exists it just so happens that there are no comments yet. So we need to separate this behaviour out from what is a valid but non existent review_id which should be an error.